oidc.go

```diff
@@ -128,16 +128,16 @@ type AuthenticationSettings struct {
 }
 
 func AuthenticationConfigFromCLI(
-	c *cli.Context, scopes []string,
+	ctx context.Context, cmd cli.Command, scopes []string,
```

Comment:
Command is a pointer in https://pkg.go.dev/github.com/urfave/cli/v3#ActionFunc, so we should not require applications to dereference it.